tests/integration/test_filter.py

```diff
@@ -1,7 +1,7 @@
 import dask.dataframe as dd
 import pandas as pd
 import pytest
-from pandas.testing import assert_frame_equal
```

Comment:
I know I said earlier to switch over to `assert_eq` here, but after learning about the default behavior of `assert_eq` in https://github.com/dask-contrib/dask-sql/pull/365#issuecomment-1068282514, I think we should probably hold off on this switch until https://github.com/dask/dask/pull/8811 is out in a Dask release and #365 is merged in 😓 would you mind switching these back to `assert_frame_equal`?